kuma/javascript/src/bcd-signal.jsx

```diff
@@ -0,0 +1,809 @@
+        );
+
+        const textAreaControl = document.createElement('textarea');
+        textAreaControl.className = 'control-input';
+        textAreaControl.id = 'supporting-material';
+        textAreaControl.maxLength = 1000;
+
+        return createFormControl({
+            header: headerText,
+            description: descriptionText,
+            el: textAreaControl,
+            index: 4,
+            optional: true
+        });
+    };
+
+    /**
+     * Builds and creates the error message box
+     * @returns Error message box as a `HTMLElement`
+     */
+    const createErrorMessageHandler = () => {
+        const errorLabelEl = document.createElement('span');
+        errorLabelEl.className = 'error-label';
+        errorLabelEl.innerHTML = gettext('Connection error:');
+
+        const errorTextEl = document.createElement('span');
+        errorTextEl.innerHTML = gettext(
+            'Sorry, we can’t seem to reach the Server. We are working to fix the problem. Please try again later.'
+        );
+
+        errorMessageWrapper = document.createElement('div');
+        errorMessageWrapper.className = 'warning error-message';
+
+        errorMessageWrapper.appendChild(errorLabelEl);
+        errorMessageWrapper.appendChild(errorTextEl);
+
+        return errorMessageWrapper;
+    };
+
+    /**
+     * Builds and creates left block with image, header and description
+     * @returns Left block as a `HTMLElement`
+     */
+    const createLeftBlock = () => {
+        const leftBlockWrapper = document.createElement('div');
+        const leftBlockImage = document.createElement('span');
+        const leftBlockHeader = document.createElement('h2');
+        const leftBlockDescription = document.createElement('p');
+        const closeButtonWrapper = document.createElement('div');
+        const closeButton = document.createElement('button');
+
+        closeButton.className = 'button close-btn';
+        closeButtonWrapper.className = 'close-button-wrapper';
+        leftBlockWrapper.className = 'column-5 left-block';
+        leftBlockHeader.className = 'left-block-header';
+        leftBlockImage.className = 'left-block-image';
+
+        closeButton.addEventListener('click', toggleBcSignalBlock);
+
+        leftBlockHeader.innerText = gettext(
+            'Tell us what’s wrong with this table'
+        );
+        leftBlockDescription.innerHTML = interpolate(
+            gettext(
+                'Our goal is to provide accurate, real values for all our compatibility data tables. Notifying MDN of inaccurate data or supplying new data pushes us further towards our goal of providing %(bStart)s100% real values%(bEnd)s to the developer community. %(br)s%(bStart)sThank you for helping.%(bEnd)s'
+            ),
+            {
+                bStart: '<b>',
+                bEnd: '</b>',
+                br: '<br />'
```

Comment:
A similar conversation was brought up above. I [responded](
https://github.com/mozilla/kuma/pull/5598#discussion_r324394375) to it shortly after that. Let me know what your thoughts are on it. Also, as you said, it may be good to discuss this item with the rest of the team to see what suggestions they have.